ptp/tools/hoppy/parser.py

```diff
@@ -93,15 +93,26 @@ def parse_report(self):
         if not transactions:
             return None
         for count, transaction in enumerate(transactions):
-            response = self._re_response.search(transaction).group().strip() + '\n\n'
-            status_code = self._re_response_status_code.findall(response)
-            parsed_response = self._re_response_parse.findall(response)
+            try:
+                response = self._re_response.search(transaction).group().strip() + '\n\n'
+            except AttributeError:
+                response = "NOT FOUND"
```

Comment:
I think that it would be better to have a sentinel value like `None` or `''` instead of a hardcoded arbitrary string.